actsync/actionsync.go

```diff
@@ -95,7 +95,6 @@ func (as *ActionSync) Stop(ctx context.Context) error {
 		return err
 	}
 	close(as.quit)
-	close(as.syncChan)
```

Comment:
do not close the channel, so it will not panic.

Reply:
better to close the channel, otherwise we are leaving a channel that may still have data (and possibly a memory leak)
can send to another chan in sync(), and wait that channel here
```
<- as.exit
```